test/jdk/java/text/Format/NumberFormat/CurrencySymbols.properties

```diff
@@ -150,7 +150,7 @@ th_TH=\u0E3F
 tr=\u00A4
 tr_TR=TL
 uk=\u00A4
-uk_UA=\u0433\u0440\u043d.
+uk_UA=\u0433\u0440\u043D.
 zh=\u00A4
 zh_CN=\uFFE5
 zh_HK=HK$
```

Comment:
Why are they not encoded into UTF-8 native?

Reply:
Not sure, thank you for catching it. Working on it right now.